lib/composer.js

```
@@ -590,5 +598,6 @@ function createDefaultAdapter (name, keyFields) {
     return result
   }
 }
+function noop () {}
```

Do we really want the default behavior to be swallowing errors? That seems like it will lead to hard to debug issues for users.

---

I agree, this does not seem like a good idea

---

should `onSubgraphError` be required?